app/graph/page.tsx

```diff
@@ -28,7 +28,9 @@ function validateGraphSelection(graphName: string): boolean {
 export default function Page() {
     const [graph, setGraph] = useState(Graph.empty());
     const [metaData, setMetaData] = useState<string[]>([]);
+    const [showData, setShowData] = useState<boolean>(true);
     const [showGraph, setShowGraph] = useState<boolean>(true);
+    const [value, setValue] = useState("graph");
```

Comment:
Suggest you'll use a better name
```suggestion
    const [selectedTab, setSelectedTab = useState("graph");
```